handbook/engineering/distribution/managed/upgrade_process.md

```diff
@@ -26,9 +26,46 @@ These videos reference some customer names, and as such can only be shared with
 
 **Note:** You do not need to watch the above videos to perform this process, these are for educational purposes only. The steps below may be more up-to-date and accurate than the videos above.
 
+## 0) Setup
+
+Managed instances configuration is tracked in [`deploy-sourcegraph-managed`](https://github.com/sourcegraph/deploy-sourcegraph-managed) - make sure you have the latest revision of this repository checked out. For basic operations like accessing an instance for these steps, see [managed instances operations](operations.md).
+
+Many of the following commands in this guide, as well as the commands [operations](./operations.md), use variables. You can replace variables with the correct values by hand, or export the appropriate values for the upgrade:
+
+```sh
+export CUSTOMER=<customer_or_instance_name>
+export VERSION=v<sourcegraph_version>
+```
+
+Start a branch for your upgrade in [`deploy-sourcegraph-managed`](https://github.com/sourcegraph/deploy-sourcegraph-managed):
+
+```sh
+git checkout -b $CUSTOMER/upgrade-to-$VERSION
+```
+
+Figure out [what instance is currently live](./operations.md#redblack-deployment-model) for the managed instance you are upgrading - it should be either `red` or `black`:
+
+```sh
+gcloud compute instances list --project=sourcegraph-managed-$CUSTOMER
+```
+
+Set up the following values (or replace them as you go):
+
+```sh
+export OLD_DEPLOYMENT=red # the current deployment
+export NEW_DEPLOYMENT=$([ "$OLD_DEPLOYMENT" = "red" ] && echo "black" || echo "red")
+```
+
+You can replace the `DEPLOYMENT` variable used in [operations](./operations.md), following the steps in this guide to determine which value `DEPLOYMENT` should be, like so:
```

Comment:
I don't understand what this line is talking about

Reply:
```suggestion
You can replace the `DEPLOYMENT` variable used in the examples in [operations](./operations.md) (using this guide to determine the appropriate value) like so:
```